### PR TITLE
Add slurm script location

### PR DESCRIPTION
**Pull request by jpa198**

### CODE REVIEW

File: scripts/submit_emod3d.py [hunk 1]
```diff
@@ -71,20 +72,21 @@ def write_sl_script(
         job_description="emod3d slurm script",
         additional_lines="#SBATCH --hint=nomultithread",
         target_host=machine,
+        write_directory=write_directory,
     )
 
     fname_slurm_script = "run_emod3d_%s_%s.sl" % (srf_name, const.timestamp)
+    fname_slurm_script = os.path.abspath(
```
Comment (jasonmotha) [inline scripts/submit_emod3d.py]: Less confusing to merge these two lines

Comment (jasonmotha) [reply scripts/submit_emod3d.py]: Why abspath?

File: shared_workflow/shared.py [hunk 2]
```diff
@@ -128,21 +128,9 @@ def generate_context(simulation_dir, template_path, **kwargs):
...
-    memory,
-    exe_time,
-    job_description,
-    partition=None,
-    additional_lines="",
-    template_path="slurm_header.cfg",
-    target_host=host,
-    mail="test@test.com",
-):
+def resolve_header(account, n_tasks, wallclock_limit, job_name, version, memory, exe_time, job_description,
```
Comment (jasonmotha) [inline shared_workflow/shared.py]: Black (as discussed)

File: scripts/submit_emod3d.py (hunk 1 above)
Comment (jpa198) [reply scripts/submit_emod3d.py]: Returns the full path from root, so things like . and ~ are expanded

**Pull request merged**